canopeum_backend/canopeum_backend/migrations/0002_alter_fertilizertype_name_alter_mulchlayertype_name_and_more.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 5.1 on 2024-10-03 18:38
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("canopeum_backend", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
migration seems to only be good for if the db is empty, is it ok? Since FK won't be found in the other table unless we add them first.

Reply:
Yeah probably better off squashing the DB again to get devs to regenerate it (we won't be able to do that once we have real data on prod, but for now it's an easy shortcut)